### DISCUSSION TITLE
Ads block warning from Indian news websites

### DISCUSSION THREAD

**Discussion by turjyasaha** [bromite/bromite#1188] [Ideas]
Dear Developer,

Please fix the issue.

Most of the Indian news websites are not openly due to ads block notification. It is opening from Brave browser and other ads block browser also.

Websites are following
timesofindia.indiatimes.com/
economictimes.com/

![Screenshot_2021-05-31-09-56-05-598_org bromite bromite](https://user-images.githubusercontent.com/24561117/120139765-ce2b6c80-c1f6-11eb-88bf-28fadedeee25.jpg)

Looking forward to hearing from you

**Comment by SuhasHebbar** [comment]
I was not able to reproduce the issue you mentioned on the mentioned websites.

I think you can handle this issue yourself though. If I understand [this changelog](https://www.bromite.org/news/2019-09-16-bromite-v77-released) correctly Bromite uses the native Chromium adblock engine which you should be able to configure per website.

![image](https://user-images.githubusercontent.com/25454994/120181819-38f69b00-c22b-11eb-8f61-77582b59a969.png)
